x-pack/plugins/ml/server/types.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { UsageCollectionSetup } from 'src/plugins/usage_collection/server';
+import { HomeServerPluginSetup } from 'src/plugins/home/server';
+import { PluginSetupContract as FeaturesPluginSetup } from '../../features/server';
+import { LicensingPluginSetup } from '../../licensing/server';
+import { SpacesPluginSetup } from '../../spaces/server';
+
+export const PLUGIN_ID = 'ml';
```

Comment:
it'd be good to move this to common as we often need this string client side. it could go in `app.ts`.
(also `API_BASE_PATH` in there can go as it's a leftover from transforms being in ML)

Reply:
Updated in 609be5c4c218d1feed4b249158c438ce7c30f99d